src/current/v24.2/changefeed-sinks.md

```diff
@@ -296,9 +296,10 @@ For a list of compatible parameters and options, refer to [Parameters]({% link {
 
 ### Pub/Sub sink configuration
 
-The `pubsub_sink_config` option allows the changefeed flushing and retry behavior of your Pub/Sub sink to be configured.
+You can configure flushing, retry, and concurrency behavior of changefeeds running to a Pub/Sink sink with the following:
```

Comment:
same re: "Pub/Sink" ?

PS aside: omggggg we sure do support a lot of versions now 😐

Reply:
I knowwwwww